packages/synapse-react-client/vite.config.ts

```diff
@@ -6,8 +6,13 @@ import { ConfigBuilder } from 'vite-config'
  */
 const config = new ConfigBuilder()
   .setIncludeReactConfig(true)
-  .setIncludeLibraryConfig(true)
-  .setBuildLibEntry(resolve(__dirname, 'src/index.ts'))
+  .setIncludeLibraryConfig(true, {
+    except: [
+      // Include @sage-bionetworks/synapse-types because the local version may drift from the version released on NPM
+      '@sage-bionetworks/synapse-types',
+    ],
+  })
+  .setBuildLibEntry(resolve(__dirname, 'src/umd.index.ts'))
```

Comment:
Update this config to use the `umd.index.ts` file, which is really "the set of synapse-react-client modules used in SWC", and has nothing to do with the UMD module format.

In the near future, I plan to make this more clear. I will update file names and remove the UMD-specific bundle scripts so we just have one build that creates ESM/CJS packages for SynapseWebClient.

For now, I'm just trying to make the changes to get this in SWC as small as possible.